Automattic/sensei
# Remove appearance customizer for Learning Mode

Resolves #6888 

We had colors customization in Appearance -> Customize for Learning Mode.
However, we didn't use those settings and it could confuse users.
In this PR, we remove the Learning Mode customizer.

## Proposed Changes
* Remove appearance customizer for Learning Mode

## Testing Instructions

1. Check Appearance menu.
2. In case you use the Course theme, there should be no Customize submenu.
3. In case you use Astra, for example, click Customize and make sure you don't see customization for Learning Mode.

## Pre-Merge Checklist

- [x] Decisions are publicly documented (p1684244809127949-slack-C02NWDZBL0H)
